quark/webreport/generate.py

```diff
@@ -0,0 +1,282 @@
+class ReportGenerator:
+    """
+    This module is for web report generating.
+    """
+
+    def __init__(self, json_report):
+
+        self.json_report = json_report
+
+        # Load html layout
+        with open("quark/webreport/upper.html") as file:
+            self.upperHTML = file.read()
+            file.close()
+
+        with open("quark/webreport/lower.html") as file:
+            self.lowerHTML = file.read()
+            file.close()
```

Comment:
The PyPi package by default doesn't include non-python files ([ref](https://setuptools.pypa.io/en/latest/userguide/datafiles.html?highlight=INclude#data-files-support)), so we may need to
+ modify the `package_data` keyword in [setup.py](https://github.com/quark-engine/quark-engine/blob/master/setup.py) to include these HTML
+ access these files by using [the recommended API](https://setuptools.pypa.io/en/latest/userguide/datafiles.html?highlight=INclude#accessing-data-files-at-runtime)

Please refer [here](https://github.com/quark-engine/quark-engine/pull/296/files) for an example.